example/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.9.16
```

Comment:
[IMO, nits] should use newer version, maybe

Reply:
Changed to use `3.10.11`